helm-maintainers-onboarding-guide.md

```diff
@@ -105,23 +105,26 @@ Core maintainers take turns triaging the issue queue. The responsibilities of a
   - If it's high severity, add it to the next milestone (patch or feature)
   - Otherwise, add it to the next MINOR release
 - Features:
-  - If it violates the SemVer rules, mark it as `Upcoming - Major` and perhaps open a dialog on whether there is an alternate way to do this that will not break our SemVer rules
+  - If it violates the SemVer rules, open a dialog on whether there is an alternate way to do this that will not break our SemVer rules
   - If it is a major feature (new subcommand, new way of doing something), ask if it can be implemented as a plugin or extension
+  - If a major change is warranted, suggest opening a [HIP](https://github.com/helm/community/blob/main/hips/hip-0001.md)
```

Comment:
I would use stronger language here. Major changes proposed for Helm 4 must go through the HIP process (see [HIP 12: "Planning"](https://github.com/helm/community/pull/179)). I would re-word this as:

> All features and breaking changes MUST be described in a HIP, and that HIP must pass the usual process (see HIP 12: "Helm 4 Development Process").